### ISSUE TITLE
Execution from symlink on linux

### ISSUE THREAD

**Issue by Gansgar** [tikzit/tikzit#80]
I wanted to install the package correctly so that I don't need to change my `$PATH`. As such I've created a symlink to link `~/.local/apps/tikzit/bin/tikzit` to  `~/.local/bin/tikzit` only to discover that the script now takes the `~/.local` folder as its root directory. 
I've now customized the startup bash file by replacing the first line, that finds the directory, with a variation that also takes symlinks into consideration. 

You can find the solution (the multiline one) in the post:
https://stackoverflow.com/a/246128/5759814

**Comment by 0xACE** [comment]
I don't understand the problem here.

What do you mean by

>only to discover that the script now takes the `~/.local` folder as its root directory.

What script are you talking about?

And I'm guessing this is the script you are referring to:

```
#!/bin/bash

SOURCE="${BASH_SOURCE[0]}"
while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
  DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
  SOURCE="$(readlink "$SOURCE")"
  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
done
DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
```


I have `tikzit` symlinked directly to the git folder, and I have had 0 issues.

**Comment by Gansgar** [comment]
Yeah. I mean that script. The version of that script that was provided with my binary wasn't aware of symlinks. I needed to change my script to the one you've in order to get it to work.

**Comment by 0xACE** [comment]
> The version of that script that was provided with my binary

I don't understand which script this is, can you link me to this so I can have a look?

I'm wondering where you got the script from and what the content looks like.
